SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/TypeStructure.cpp

```diff
@@ -259,6 +259,42 @@ TSharedPtr<FUnrealType> CreateUnrealTypeInfo(UStruct* Type, uint32 ParentChecksu
 		return TypeNode;
 	}
 
+	if (Class->IsChildOf<AActor>())
+	{
+		const AActor* CDO = Cast<AActor>(Class->GetDefaultObject());
+
+		// Handle components attached to the actor that don't have a property pointing to them.
+		for (UActorComponent* Component : CDO->GetComponents())
```

Comment:
Super unhappy about this code since this will overlap with any subobjects that have properties pointing to them.

Reply:
Yeah this does seem problematic. Can we strip actor components from the properties?

Reply:
This shouldn't be a huge issue since we de-dupe them in `GetAllSubobjects`.